packages/entities/entities-upstreams-targets/src/components/TargetForm.vue

```diff
@@ -104,6 +118,8 @@ const emit = defineEmits<{
   (e: 'cancel'): void
 }>()
 
+const failoverEnabled = inject(FAILOVER_INJECTION_KEY, true)
```

Comment:
Are we using injection instead of a prop because the `TargetForm` component is not intended to be used standalone in the consuming app?

I think we could still make this a prop in both `TargetList` and `TargetForm`. In `TargetList`, the prop value would be determined by the consuming app, while in `TargetForm` it could be set [here](https://github.com/Kong/public-ui-components/blob/380f06f74148c26795da5302d3a24f58a8897186/packages/entities/entities-upstreams-targets/src/components/TargetsList.vue#L118) directly.

Keeping it as a prop and documenting it in the README would be more consistent with how we handle other switches.